### DISCUSSION TITLE
Inconsistent column index between "Properties" and "Metadata"

### DISCUSSION THREAD

**Discussion by nikiromagnoli** [dbeaver/dbeaver#14690] [General]
#### System information: 
- Operating system: Ubuntu 20.04.3 LTS
- DBeaver version:   CE 21.2.5.202111141807
- Additional extensions: none

#### Connection specification:
- Database name and version: MariaDB Ver 15.1 Distrib 10.5.8-MariaDB
- Driver name: MariaDB
- Do you use tunnels or proxies (SSH, SOCKS, etc)? No

#### Describe the problem you're observing:
Columns are numbered in tables (index). In DBeaver column number is available from table's properties tab and "Metadata" panel.

The problem is that while in "Properties" numbers are 1-indexed

![immagine](https://user-images.githubusercontent.com/5136855/143275139-2bab761b-58ee-4a4c-acf9-fa64c3729780.png)

in "Metadata" panel they are 0-indexed

![immagine](https://user-images.githubusercontent.com/5136855/143275221-e57da9be-80c3-47a3-ad45-9d3a8a6e202d.png)

This results in numbers being offsetted by 1 between panes. Is it intended behaviour?

#### Steps to reproduce, if exist:
Just:
1. create a table
2. add a column
3. watch index in "Properties" tab
4. watch index in "Metadata" panel

Thank you for your time and this great tool.

**Comment by ShadelessFox** [comment]
Hello @TechNyquist,

The <kbd>Metadata</kbd> tab shows data reported by the JDBC driver. In <kbd>Columns</kbd> tab, they're 1-indexed because it feels more natural.
